recipe/meta.yaml

```diff
@@ -22,6 +22,9 @@ test:
   imports:
     - pyct
     - pyct.cmd
+  requires:
+    - flake8
+    - pytest
```

Comment:
It is OK to leave these here but we only need to add those if we are going to run the tests, otherwise don't worry about test-requires.